log/handlers.ts

```diff
@@ -42,7 +42,7 @@ export class BaseHandler {
       const value = logRecord[p1 as keyof LogRecord];
 
       // do not interpolate missing values
-      if (value == null) {
+      if (value === null) {
```

Comment:
This looks like a check for missing property of the object. I think this should compare with `undefined`
```suggestion
      if (value === undefined) {
```